e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -0,0 +1,711 @@
+        .replace(`{{${COUNT_COLUMN_ID}}}`, POINT_COUNT)
+        .replace(`{{${CREATED_AT_COLUMN_ID}}}`, POINT_CREATED_AT)
+        .replace(`{{${DASHBOARD_FILTER_TEXT.slug}}}`, FILTER_VALUE);
+
+      const dashboardDetails = {
+        parameters: [DASHBOARD_FILTER_TEXT],
+      };
+
+      cy.createQuestionAndDashboard({ questionDetails, dashboardDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Go to a custom destination").click();
+      cy.get("aside").findByText("URL").click();
+      modal().findByText("Values you can reference").click();
+      popover().within(() => {
+        cy.findByText(COUNT_COLUMN_ID).should("exist");
+        cy.findByText(CREATED_AT_COLUMN_ID).should("exist");
+        cy.findByText(DASHBOARD_FILTER_TEXT.name).should("exist");
+        cy.realPress("Escape");
+      });
+      modal().within(() => {
+        cy.findByRole("textbox").type(escapedUrlWithParams);
+        cy.button("Done").click();
+      });
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      cy.findByTestId("field-set").click();
+      popover().within(() => {
+        cy.findByPlaceholderText("Enter some text").type(FILTER_VALUE);
+        cy.button("Add filter").click();
+      });
+
+      onNextAnchorClick(anchor => {
+        expect(anchor).to.have.attr("href", expectedUrlWithParams);
+        expect(anchor).to.have.attr("rel", "noopener");
+        expect(anchor).to.have.attr("target", "_blank");
+      });
+      clickLineChartPoint();
+    });
+
+    it("does not allow updating dashboard filters if there are none", () => {
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside")
+        .findByText("Update a dashboard filter")
+        .invoke("css", "pointer-events")
+        .should("equal", "none");
+    });
+
+    it("allows updating single dashboard filter", () => {
+      const dashboardDetails = {
+        parameters: [DASHBOARD_FILTER_TEXT],
+      };
+
+      cy.createQuestionAndDashboard({ questionDetails, dashboardDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Update a dashboard filter").click();
+      addTextParameter();
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      clickLineChartPoint();
+      cy.findAllByTestId("field-set")
+        .should("have.length", 1)
+        .should("contain.text", POINT_COUNT);
+      cy.location("search").should(
```

Comment:
To ensure we're still staying on the same dashboard and only updating the filter, should we add an assertion for either
- dashboard name
- or the dashboard ID in the URL like in other test case

Reply:
010804b743fbb99a952a5cd472664bf172cafe9a